pygmt/tests/test_grdfilter.py

```diff
@@ -0,0 +1,101 @@
+"""
+Tests for grdfilter.
+"""
+import os
+
+import numpy as np
+import numpy.testing as npt
+import pytest
+import xarray as xr
+from pygmt import grdfilter, grdinfo
+from pygmt.datasets import load_earth_relief
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+
+
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_earth_relief(registration="pixel")
+
+
+def test_grfilter_dataarray_in_dataarray_out(grid):
+    """
+    grdfilter an input DataArray, and output as DataArray.
+    """
+    result = grdfilter(grid=grid, filter="g600", distance="4")
+    # check information of the output grid
+    assert isinstance(result, xr.DataArray)
+    assert result.coords["lat"].data.min() == -89.5
+    assert result.coords["lat"].data.max() == 89.5
+    assert result.coords["lon"].data.min() == -179.5
+    assert result.coords["lon"].data.max() == 179.5
+    np.testing.assert_almost_equal(result.data.min(), -6147.47265625, decimal=2)
+    np.testing.assert_almost_equal(result.data.max(), 5164.1157, decimal=2)
```

Comment:
```suggestion
    npt.assert_almost_equal(result.data.min(), -6147.47265625, decimal=2)
    npt.assert_almost_equal(result.data.max(), 5164.1157, decimal=2)
```

Reply:
Hoping I don't cause an issue, but I'll commit this, but it looks like it was already changed to `npt` in 1b0dc75

Reply:
`np.testing.assert_almost_equal` is used in two tests, and 1b0dc75ccdae933cc32e5933b673621957570f85 only changed the 2nd one.

Reply:
Whoops! Just forgetfulness on my part.